plugins/generator-1.14.4/forge-1.14.4/procedures/entity_get_shootpower.java.ftl

```diff
@@ -0,0 +1 @@
+((projectileentity != null)?((projectileentity.getEntity() instanceof ${generator.map(field$projectiles, "projectiles")})?(((${generator.map(field$projectiles, "projectiles")}) projectileentity).getMotion().distanceTo(Vec3d.ZERO)):0):0)
```

Comment:
You don't need to request projectile type here, just cast to IProjectile.

Reply:
Done! Last commit changed it.